cmake/linux/winegcc_wrapper.in

```diff
@@ -0,0 +1,63 @@
+#!/bin/sh
+# Wrapper script for winegcc to remove .exe file ending automatically
+# appended by winebuild.
+# Usage: winegcc <args ...>
```

Comment:
Your usage says `winegcc`.  I'm sure this was intentional, but more accurately...

```bash
# Usage: winegcc_wrapper <args ...>

# --- OR ---
# Usage:
#    CONFIGURE_FILE("path/to/winegcc_wrapper.in winegcc_wrapper @ONLY)
#    SET(WINEGCC "path/to/winegcc_wrapper")
#    winegcc <args ...>
```

Probably nitpicking, but didn't know if it was a typo or intentional. :)